packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx

```diff
@@ -229,6 +243,7 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
                                 item,
                             });
                         }}
+                        sx={{ display: isSqlChart ? 'none' : 'block' }}
```

Comment:
could we use the `hidden` prop instead? 
```tsx
hidden={isSqlChart}
```

otherwise we have to deal with default styling if it's not a `sql chart`

<img width="420" alt="image" src="https://github.com/user-attachments/assets/55b604aa-ecdf-4c61-a39c-f39a14df939e">

Reply:
hidden didn't work to me, let me try again

Reply:
then we can do `disabled`?

Reply:
https://github.com/mantinedev/mantine/issues/5521#issuecomment-1878444848

Reply:
disabled without tooltip can be confusing, don't you think ? and I don't want to add many extra tooltips...

Reply:
I can do this instead 
`                            sx={isSqlChart ?{ display:  'none'}: {}}`

Reply:
then we can replace in the ternary the `block` with `flex` 👍

Reply:
no need to add any flex, we can leave it empty 

> sx={isSqlChart ?{ display:  'none'}: {}}


![image](https://github.com/user-attachments/assets/9de82ba4-9f49-41d5-9306-21692e5efc87)